boardgamegeek/api.py

```diff
@@ -874,7 +874,7 @@ def game(self, name=None, game_id=None, choose=BGGChoose.FIRST, versions=False,
                                     game_id=game_id,
                                     html_parser=html_parser)
 
-        if not comments:
+        if not (comments or rating_comments):
```

Comment:
This is needed for any ratings/comments to be returned when `rating_comments=True`